tests/ca_test.go

```diff
@@ -0,0 +1,79 @@
+// +build apitests
```

Comment:
@plorenz there are some testing facilities that you have built that I am not sure I am taking advantage of. Let me know if anything jumps out.